dataline-acceptance-tests/src/accTest/java/io/dataline/tests/acceptance/AcceptanceTests.java

```diff
@@ -0,0 +1,110 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.dataline.tests.acceptance;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.dataline.api.model.SourceIdRequestBody;
+import io.dataline.api.model.SourceImplementationCreate;
+import io.dataline.api.model.SourceImplementationRead;
+import io.dataline.api.model.SourceReadList;
+import io.dataline.api.model.SourceSpecificationRead;
+import io.dataline.commons.json.Jsons;
+import io.dataline.config.persistence.PersistenceConstants;
+import java.io.IOException;
+import java.util.UUID;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class AcceptanceTests {
+
+  static final MediaType JSON_CONTENT = MediaType.get("application/json; charset=utf-8");
+  static final String SERVER_URL = "http://localhost:8001/api/v1/";
+  static final OkHttpClient HTTP_CLIENT = new OkHttpClient();
+
+  static PostgreSQLContainer PSQL_DB1;
+
+  @BeforeAll
+  public static void init() {
+    PSQL_DB1 = new PostgreSQLContainer();
+    PSQL_DB1.start();
+  }
+
+  @Test
+  public void testCreateSourceImplementation() throws IOException {
+    UUID postgresSourceId = getPostgresSourceId();
+    SourceSpecificationRead sourceSpecRead =
+        callApi("source_specifications/get", new SourceIdRequestBody().sourceId(postgresSourceId), SourceSpecificationRead.class);
+
+    ObjectNode dbConfiguration = new ObjectNode(JsonNodeFactory.instance)
+        .put("host", PSQL_DB1.getHost())
+        .put("password", PSQL_DB1.getPassword())
+        .put("port", PSQL_DB1.getFirstMappedPort())
+        .put("dbname", PSQL_DB1.getDatabaseName())
+        .put("filter_dbs", PSQL_DB1.getDatabaseName())
+        .put("user", PSQL_DB1.getUsername());
+
+    UUID defaultWorkspaceId = PersistenceConstants.DEFAULT_WORKSPACE_ID;
+    UUID sourceSpecificationId = sourceSpecRead.getSourceSpecificationId();
+
+    SourceImplementationCreate sourceImplementationCreate = new SourceImplementationCreate().sourceSpecificationId(sourceSpecificationId)
+        .workspaceId(defaultWorkspaceId)
+        .connectionConfiguration(dbConfiguration);
+
+    SourceImplementationRead createResponse = callApi("source_implementations/create", sourceImplementationCreate, SourceImplementationRead.class);
```

Comment:
I wonder if we could generate a client from the oas definition. that will also help catch name changes if they happen

Reply:
Will look into this

Reply:
This hasn't been 100% straightforward, going to do this in a following PR so Jared can work off of this module